src/components/Reminder.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
+  import { Alert } from "sveltestrap/src";
+
+  export let card;
+
+  let dueDate = new Date(card.due_date_time);
+</script>
+
+{#if !card.is_archived}
+  {#if Date.now() >= new Date(card.remind_date_time)}
```

Comment:
Maybe combine both `if` statements? Or we could keep it as is for readability.

Reply:
I would prefer keep it as is for readability. Thank you for the review, sir.